source/_integrations/voicerss.markdown

```diff
@@ -34,15 +34,30 @@ codec:
   description: The audio codec.
   required: false
   type: string
-  default: mp3
+  default: " mp3 "
 format:
   description: The audio sample format.
   required: false
   type: string
-  default: 8khz_8bit_mono
+  default: " 8khz_8bit_mono "
+voice:
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

Reply:
I wish there was proper documentation for to how to create a config flow, with 'Hello world' -style samples etc. The documentation linked is too vague for a beginner.